src/main.rs

```diff
@@ -31,22 +27,38 @@ fn content(content_type: &str, reply: impl Reply) -> impl Reply {
 
 // #[get("/")]
 fn index() -> impl Reply {
-    content("text/html; charset=utf-8", INDEX_HTML)
+    #[cfg(not(debug_assertions))]
```

Comment:
I'm surprised that this is the preferred way to check what build mode we're in, but there you go

Reply:
As was I!